File: app/src/main/java/com/rockspin/androiddevtest/APIService.java
package com.rockspin.androiddevtest;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface APIService {

    /**
     * Perform a call to fetch a list of Extra-Vehicular Cosmonaut Activity starting from 1965
     * @return The Retrofit Call
     */
    @GET("9kcy-zwvn.json?$order=date&$limit=20")
    Call<List<CosmonautActivity>> getEVList();
}

File: app/src/main/java/com/rockspin/androiddevtest/APIService.kt
package com.rockspin.androiddevtest

import retrofit2.Call
import retrofit2.http.GET

interface APIService {

    /**
     * Perform a call to fetch a list of Extra-Vehicular Cosmonaut Activity starting from 1965
     * @return The Retrofit Call
     */
    @get:GET("9kcy-zwvn.json?\$order=date&\$limit=20")
    val evList: Call<List<CosmonautActivity>>
}
